django/db/backends/mysql/operations.py

```diff
@@ -141,7 +141,7 @@ def last_executed_query(self, cursor, sql, params):
         # With MySQLdb, cursor objects have an (undocumented) "_last_executed"
         # attribute where the exact query sent to the database is saved.
         # See MySQLdb/cursors.py in the source distribution.
-        query = getattr(cursor, '_last_executed', None)
+        query = getattr(cursor, '_executed', None)
```

Comment:
Should we lookup both attributes until we drop support for `mysqlclient<1.3.14`?

Reply:
`_executed` also exists in older versions. I tested with Django's documented minimum, mysqlclient 1.3.7.